csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests/runtest.sh

```diff
@@ -24,6 +24,7 @@ if [ $RunTestCsharp = "true" ]; then
     OPSET_VERSION=$(grep "${ONNX_VERSION_NUMBER}" "${ONNX_DIR}/docs/Versioning.md" | sed -E "s/${ONNX_VERSION_NUMBER}\|[^|]+\|([0-9]+)\|.*/\1/")
     mkdir -p "${BUILD_BINARIESDIRECTORY}/models"
     ln -s "${ONNX_DIR}/onnx/backend/test/data/node" "${BUILD_BINARIESDIRECTORY}/models/opset${OPSET_VERSION}"
+    export ORT_CSHARP_TEST_ONNX_MODEL_ROOT_PATH="${BUILD_BINARIESDIRECTORY}/models"
```

Comment:
should this be set for non-MacOS too?

Reply:
Maybe not because

- Different OS has their own ways to set model path.
- This file is only called from test_linux.yml and test_macos.yml. ![image](https://user-images.githubusercontent.com/3524474/178079667-5227ce9f-ed61-4134-92ec-fc5dde856810.png)

Reply:
We should use the script provided in PR #12220 to download test models from onnx git repo, instead of "ln -s ..."